dev-utils/visualise-ui-hierarchy/README.md

```diff
@@ -0,0 +1,59 @@
+# Visualise (the) UI hierarchy (of polyPod features)
+
+## Requirements
+
+Aside from the stuff you probably already have (Node, Make, Python and Git), you
```

Comment:
Configuration is always better in the environment... Thus include them in the Makefile as preconditions.